client/blocks/reader-combined-card/index.jsx

```diff
@@ -118,9 +118,9 @@ class ReaderCombinedCard extends React.Component {
 						) }
 				</header>
 				<ul className="reader-combined-card__post-list">
-					{ posts.map( post => (
+					{ posts.map( ( post, i ) => (
 						<ReaderCombinedCardPost
-							key={ `post-${ post.ID }` }
+							key={ `post-${ i }-${ post && post.ID }` }
```

Comment:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
If the list is reordered when accepting updates it'll be suboptimal but it's better than any alternative we could easily throw in here. At this stage a post can be undefined so we can't fully rely on ID

Reply:
To clarify: the biggest issue that you want to prevent is having two different components presenting with the same key.  This is particularly likely to happen when components are reordered.

Since the index is only part of a composite key that includes the `id`, a reordering will be less efficient (because it cannot reuse the same components and move them, it has to start from scratch) but shouldn't be a bug.

Reply:
do we pass through the postKey? that should have something.

Reply:
I'll check, that'd probably work 👍